contracts/SmartMeter.sol

```diff
@@ -3,6 +3,7 @@
 pragma solidity ^0.8.9;
 
 contract SmartMeter {
+    bytes32 hashedValue;
     address owner;
```

Comment:
Skal vi ikke fjerne owner helt fra SmartMeter så?

Reply:
Jep

Reply:
Vi kan ikke fjerne owner, owner er smart meterets egen key og ikke brugeren. Vi kan selfølgelig godt lave samme finte med smart meteret, nu ved vi jo hvordan man gør.

Reply:
owner bliver sat i konstruktøren med owner = msg.sender;, så det er bare Wallet'en til den der deployer. Men ja vi bruger en persons wallet som key til den adresse i CableCompany, men selve owner bruger vi vel ikke

Reply:
Men er det ikke smart meter selv der deployer? Hvis den bliver deployed fra owner kan man jo lave fake logs og så holder vores Trusted Computing assumption ikke

Reply:
Jo det kan vi godt antage. Lad os bare lade den blive der så

Reply:
Vi har også brug for at kunne køre require sender == owner i createLogs, ellers ville alle kunne lave logs og opdatere batteri niveauet